index.html

```diff
@@ -209,24 +209,30 @@ <h1 class="contact-description">
           chat please get in touch.
         </h1>
```

Comment:
- [ ] You are supposed to have only one `h1` tag in your entire project, as required by HTML best practice, please change the other one to `h2`... `h6`

Reply:
Hi, according to this from a Microverse lesson http://diveintohtml5.it/semantics.html#:~:text=And%20in%20HTML5%2C%20each%20section%20can%20have%20its%20own%20%3Ch1%3E%20element.
in HTML5 each section can have its own h1 element